app/views/handlers/__init__.py

```diff
@@ -8,4 +13,17 @@
     "IndividualResponseHandler",
     "IndividualResponseHowHandler",
     "IndividualResponsePostAddressConfirmHandler",
+    "individual_response_url"
 ]
+
+
+def individual_response_url(schema: Mapping, list_item_id: str, questionnaire_store: QuestionnaireStore) -> Union[str, None]:
+    if "individual_response" in schema.json:
+        for_list = schema.json["individual_response"]["for_list"]
+
+        if list_item_id != questionnaire_store.list_store._lists[for_list].primary_person:
+            return url_for(
+                "individual_response.request_individual_response",
+                list_item_id=list_item_id,
+            )
```

Comment:
This would need to pass a query param to handle where the previous link goes to. Currently clicking previous goes to the introduction page but it should return to the remove screen.